packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -161,12 +162,22 @@ async function transformerVersionCheck(context, resourceDir, cloudBackendDirecto
   // if we already asked the confirmation question before at a previous push
   // or during current operations we should not ask again.
   const showPrompt = !(cloudVersionExist || localVersionExist);
+  if (!showPrompt) {
+    transformerVersion = 3;
```

Comment:
Why do we go to version 3 here?

Reply:
This should be the version that we are currently at. What does TRANSFORM_CURRENT_VERSION do?

Reply:
Can we add unit tests testing 
1. No transform conf version
2. Transform conf version set to 3
3. Transform conf version set to current version
3. Transform conf set to 100